src/common.speech/IntentServiceRecognizer.ts

```diff
@@ -42,22 +41,20 @@ export class IntentServiceRecognizer extends ServiceRecognizerBase {
         connectionFactory: IConnectionFactory,
         audioSource: IAudioSource,
         recognizerConfig: RecognizerConfig,
-        recognizer: IntentRecognizer,
-        intentDataSent: boolean) {
+        recognizer: IntentRecognizer) {
         super(authentication, connectionFactory, audioSource, recognizerConfig, recognizer);
         this.privIntentRecognizer = recognizer;
-        this.privIntentDataSent = intentDataSent;
+        this.privIntentDataSent = false;
     }
 
     public setIntents(addedIntents: { [id: string]: AddedLmIntent; }, umbrellaIntent: AddedLmIntent): void {
         this.privAddedLmIntents = addedIntents;
         this.privUmbrellaIntent = umbrellaIntent;
+        this.privIntentDataSent = true;
```

Comment:
>this.privIntentDataSent = true; [](start = 8, length = 31)

why is this set to true?

Reply:
The goal behind the variable was to see if the caller had added any intents to the recognizer or not.